src/main/java/ch/njol/skript/events/EvtJump

```diff
@@ -0,0 +1,39 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser;
+import org.bukkit.Statistic;
+import org.bukkit.event.Event;
+import org.bukkit.event.player.PlayerStatisticIncrementEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+public class EvtJump extends SkriptEvent {
```

Comment:
You should have documentation annotations on the class opposed to the methods c:

Reply:
Since it's a SkriptEvent they don't need to go on the class :)